libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/dashboardDropdownBody/configuration/limitValues/dialog/ActionSelectionPage.tsx

```diff
@@ -0,0 +1,92 @@
+// (C) 2024 GoodData Corporation
+
+import React, { ReactNode } from "react";
+import { useIntl } from "react-intl";
+import { Typography, Separator } from "@gooddata/sdk-ui-kit";
+
+import { messages } from "../../../../../../../locales.js";
+
+import { PopupHeader } from "./PopupHeader.js";
+import { useDashboardUserInteraction } from "../../../../../../../model/index.js";
+
+interface IActionProps {
+    title: React.ReactNode;
+    description: React.ReactNode;
+    onClick: () => void;
+}
+
+const Action: React.FC<IActionProps> = ({ title, description, onClick }) => {
+    return (
+        <div className="attribute-filter__limit__popup__action-select__option" onClick={onClick}>
+            <div>
+                <Typography tagName="h3" className="attribute-filter__limit__popup__action-select__title">
+                    {title}
+                </Typography>
+                <p className="attribute-filter__limit__popup__action-select__description">{description}</p>
+            </div>
+            <div className="gd-icon-chevron-right attribute-filter__limit__popup__action-select__icon" />
+        </div>
+    );
+};
+
+export interface IActionSelectionPageProps {
+    onAddFilters: () => void;
+    onAddLimitingItems: () => void;
+    onClose: () => void;
+}
+
+export const ActionSelectionPage: React.FC<IActionSelectionPageProps> = ({
+    onAddFilters,
+    onAddLimitingItems,
+    onClose,
+}) => {
+    const intl = useIntl();
+    const { attributeFilterInteraction } = useDashboardUserInteraction();
+
+    const onChooseParentFilters = () => {
+        onAddFilters();
+        attributeFilterInteraction("attributeFilterLimitAddParentFilterOptionClicked");
+    };
+
+    const onChooseMetrics = () => {
+        onAddLimitingItems();
+        attributeFilterInteraction("attributeFilterLimitAddMetricOptionClicked");
+    };
+
+    return (
+        <>
+            <PopupHeader title={intl.formatMessage(messages.filterAddItemTitle)} onClose={onClose} />
+            <div className="attribute-filter__limit__popup__content">
+                <Action
+                    onClick={onChooseParentFilters}
+                    title={intl.formatMessage({ id: "attributesDropdown.valuesLimiting.addFilters.title" })}
+                    description={intl.formatMessage({
+                        id: "attributesDropdown.valuesLimiting.addFilters.description",
+                    })}
+                />
+                <Separator />
+                <Action
+                    onClick={onChooseMetrics}
+                    title={intl.formatMessage({
+                        id: "attributesDropdown.valuesLimiting.addLimitingItem.title",
+                    })}
+                    description={intl.formatMessage(
+                        { id: "attributesDropdown.valuesLimiting.addLimitingItem.description" },
+                        {
+                            a: (chunk: ReactNode) => (
+                                <a
```

Comment:
not sure if this is the case too, but usually GD Help links are not shown for whitelabeling, eg. 
https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/CreatableAttributeFilter.tsx#L41

Reply:
Good point, fixed. My code removes the whole "Learn more" part if white labeling is enabled.